app/models/inventory-batch.js

```diff
@@ -16,7 +16,7 @@ export default AbstractModel.extend({
     },
     inventoryItemTypeAhead: {
       presence: {
-        if(object) {
+        unless(object) {
```

Comment:
Glad to hear I didn't mess something up. I worry with these pull requests I'll pull you away from more pressing matters if I start introducing bugs and other issues. Thanks as always.